currency-exchange-service/src/main/java/com/ubaid/ms/currencyexchangeservice/task/PopulateDBWithExchangeRates.java

```diff
@@ -34,7 +34,6 @@
     private final Config config;
     private final ExchangeRateService exchangeRateService;
 
-
     @Autowired
     @Lazy
     public PopulateDBWithExchangeRates(RestTemplate restTemplate,
```

Comment:
check if we can add @RequiredArg annotation to replace this consturctor.

Reply:
I have changed the constructor to @RequiredArg, but all tests are failing. After adding @Lazy along with @RequiredArg, test are passing and the application is running but, the task is not triggering itself. So, I guess, the current implementation is good.